torch/_functorch/aot_autograd.py

```diff
@@ -3708,11 +3726,13 @@ def aot_module_simplified(
         no_tangents=False,
     )
 
-    compiled_fn = create_aot_dispatcher_function(
-        functional_call,
-        full_args,
-        aot_config,
-    )
+    from torch._dynamo import compiled_autograd
```

Comment:
Why not import at the top?

Reply:
Dynamo imports aot_autograd.py, would be circular.

Reply:
Well:
- _dynamo init doesn't import compiled_autograd. It should?
- circular import in python are ok as long as the other piece is already there. Like if you import compiled_autograd in dynamo init before importing aot_autograd.
- You can skip the import here and use torch._dynamo.compiled_auotgrad I think?

But that's minor